README.md

```diff
@@ -38,6 +42,20 @@ Join our Telegram group chat here: https://t.me/onzecorona or reach out on konta
 
 Click on "Issues" in the menu above to see what we need help with.
 
+## Adding a new language
+
+Adding a new language should be pretty straightforward. If you need help, you can always ask in the Telegram group chat or contact us by email. The following is needed in order to set up a new language:
+
+- Add language-specific fields to the config
+- In `app/locales` you have to add (follow filename convention of the files that are already there):
+  - A word list that is used for generating unique profile links. If the word list contains between 1000 and 10000 words, you should set `PASSCODE_LENGTH: 4` in the config. If it contains more than 10000 words, `PASSCODE_LENGTH: 3` should be sufficient.
+  - List of municipalities.
```

Comment:
Should probably elaborate on this one and the next.

Reply:
I would change the header to "Adding a new language" -> "How to launch the site in your country".